ganga/GangaDirac/__init__.py

```diff
@@ -26,20 +25,34 @@
     if "GANGADIRACENVIRONMENT" in os.environ:
         diracenv = os.environ["GANGADIRACENVIRONMENT"]
 
-    configDirac.addOption('DiracEnvFile', diracenv,
-                          'DEPRECATED. Ganga environment file for DIRAC environment (do not change unless you are sure you know what you are doing).')
+    configDirac.addOption(
+        'DiracEnvFile',
+        diracenv,
+        ('DEPRECATED. Ganga environment file for DIRAC environment'
```

Comment:
I can see that the autopep8 is line wrapping here but within the brackets () of the addOption I don't think it needs the additional brackets () around the line wrapped text string.

Having said that it's probably too much effort to unpick this one change everywhere whilst keeping the rest so I think we can live with it.